packages/driver/src/cy/commands/origin/index.ts

```diff
@@ -234,18 +198,14 @@ export default (Commands, Cypress: Cypress.Cypress, cy: Cypress.cy, state: State
                   duringUserTestExecution: Cypress.state('duringUserTestExecution'),
                   hookId: Cypress.state('hookId'),
                   originCommandBaseUrl: location.href,
-                  parentOriginPolicies: [cy.getRemoteLocation('originPolicy')],
                   isStable: Cypress.state('isStable'),
-                  autOrigin: Cypress.state('autOrigin'),
+                  autLocation: Cypress.state('autLocation').href,
```

Comment:
is it possible `autLocation` here is ever undefined?

Reply:
not in all my testing, but it wouldn't hurt to hedge our bets by using optional chaining.